### ISSUE TITLE
setget syntax warning false positive

### ISSUE THREAD

**Issue by cliffrowley** [godotengine/godot-vscode-plugin#105]
Just to add to the other false positives that have been discovered in the syntax checker, the same applies to gdscript's setget feature:

```
# Warning: foobar is never used
export(String) var foobar setget set_foobar, get_foobar

func set_foobar(val):
  # do something

func get_foobar():
  # return something
```

(Edited to remove plain export example, which actually makes sense since the exported variable should still be referenced by the script - setget however likely won't and would instead use a backing variable or proxy the value elsewhere).

**Comment by Geequlim** [comment]
The new version fixed the problem and will be published when godot 3.2 RC release.

**Comment by Geequlim** [comment]
I'm closing this as the new version has fixed this issue.
